src/frontend/components/UI/Sidebar/index.css

```diff
@@ -0,0 +1,140 @@
+.Sidebar {
+  --sidebar-horizontal-padding: var(--space-md);
+  --sidebar-vertical-padding: var(--space-lg);
+  --sidebar-width: 350px;
+  --sidebar-background: var(--navbar-background);
+  --sidebar-divider-color: var(--divider);
+
+  width: var(--sidebar-width);
+  position: relative;
+  z-index: 10;
+  display: grid;
+  grid-template-rows: min-content 2fr min-content min-content;
+  grid-template-areas: 'icon' 'links' 'utils' 'updates' 'version';
+  grid-area: sidebar;
+  padding: 0;
+  font-family: var(--secondary-font-family);
+  text-align: start;
+  background: var(--sidebar-background);
+  overflow: auto;
+  overflow-x: hidden;
+  -webkit-app-region: drag;
+
+  .heroicIcon {
+    grid-area: icon;
+    width: 50px;
+    height: 68px;
+    margin-top: var(--space-xs);
+    justify-self: center;
+  }
+
+  .SidebarLinks {
+    grid-area: links;
+  }
+
+  .heroicVersion {
+    grid-area: version;
+    margin-block: 0 var(--sidebar-vertical-padding);
+    margin-inline: var(--sidebar-horizontal-padding) var(--space-3xs);
+    -webkit-app-region: no-drag;
+  }
```

Comment:
I'm still keeping these types of properties here in the parent and not in the component's css because these styles should be controlled by the parent since they are styles "outside" the component